pages/ChatList/index.tsx

```diff
@@ -29,106 +30,28 @@ const ChatList: React.FunctionComponent<Props> = ({
-          timeStamp={new Date('2020/01/30')}
-          lastMessage={"Hey there!!!"}
-        />
-        <ChatItem
-          userImageSource={{ uri: "https://picsum.photos/200" }}
-          userName="Vikas"
-          timeStamp={new Date()}
-          lastMessage={"Hey there!!!"}
-        />
-        <ChatItem
-          userImageSource={{ uri: "https://picsum.photos/200" }}
-          userName="Aman"
-          timeStamp={new Date()}
-          lastMessage={"Hey there!!!"}
-        />
-        <ChatItem
-          userImageSource={{ uri: "https://picsum.photos/200" }}
-          userName="Manish"
-          timeStamp={new Date()}
-          lastMessage={"Hey there!!!"}
-        />
-        <ChatItem
-          userImageSource={{ uri: "https://picsum.photos/200" }}
-          userName="Love"
-          timeStamp={new Date()}
-          lastMessage={"Hey there!!!"}
-        />
-        <ChatItem
-          userImageSource={{ uri: "https://picsum.photos/200" }}
-          userName="Manoj"
-          timeStamp={new Date()}
-          lastMessage={"Hey there!!!"}
-        />
-        <ChatItem
-          userImageSource={{ uri: "https://picsum.photos/200" }}
-          userName="Swati"
-          timeStamp={new Date()}
-          lastMessage={"Hey there!!!"}
-        />
-        <ChatItem
-          userImageSource={{ uri: "https://picsum.photos/200" }}
-          userName="Vikas"
-          timeStamp={new Date()}
-          lastMessage={"Hey there!!!"}
-        />
-        <ChatItem
-          userImageSource={{ uri: "https://picsum.photos/200" }}
-          userName="Aman"
-          timeStamp={new Date()}
-          lastMessage={"Hey there!!!"}
-        />
-        <ChatItem
-          userImageSource={{ uri: "https://picsum.photos/200" }}
-          userName="Manoj"
-          timeStamp={new Date()}
-          lastMessage={"Hey there!!!"}
-        />
-        <ChatItem
-          userImageSource={{ uri: "https://picsum.photos/200" }}
-          userName="Swati"
-          timeStamp={new Date()}
-          lastMessage={"Hey there!!!"}
-        />
-        <ChatItem
-          userImageSource={{ uri: "https://picsum.photos/200" }}
-          userName="Vikas"
-          timeStamp={new Date()}
-          lastMessage={"Hey there!!!"}
-        />
-        <ChatItem
-          userImageSource={{ uri: "https://picsum.photos/200" }}
-          userName="Aman"
-          timeStamp={new Date()}
-          lastMessage={"Hey there!!!"}
-        />
+        {
+          chatList.map((data, indexKey) => {
+            let checkUser = data.users.filter(({ isSameUser }) => isSameUser == false);
+            return <View key={indexKey}>
+              {
+                checkUser.map((users, index) => {
+                  return <>
+                    <ChatItem
+                      key={index}
+                      userImageSource={{ uri: users.image }}
+                      userName={users.name}
+                      timeStamp={new Date()}
```

Comment:
timestamp should also come from constants

Reply:
I have resolved the issue